launchlib/const.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package launchlib
+
+import (
+	"fmt"
+)
+
+const (
+	OutputFileName   = "startup.log"
+	OutputFileFormat = "var/log/%s" + OutputFileName
```

Comment:
This is still not actually the `OutputFileFormat`, the output file format is `var/log/%s-startup.log`.  The `PrimaryOutputFile` is then different, and doesn't actually conform to this format.

Reply:
Can you clarify what you're getting at here?

Reply:
The main process start up log gets written to `var/log/startup.log`, the format of sub-processes is then `var/log/%s-startup.log`.  I don't think the format should be `var/log/%sstartup.log` where the parameter is not the name of the sub-process, but actually `name + "-"` since it's very non-obvious the dash would be added later.